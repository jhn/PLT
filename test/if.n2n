fn if_test() -> Void {
	if (true) {
		print("success")

	}
	else{

	}

	if(4<5){
		print(6)
		if(0<1){
			print(7)

		}
		else{
		
		}
	}
}