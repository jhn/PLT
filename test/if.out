success
6
7
